.rubocop.yml

```diff
@@ -27,6 +27,10 @@ Metrics/ClassLength:
 Metrics/MethodLength:
     Enabled: false
 
+Metrics/ModuleLength:
```

Comment:
I disabled this cop for all specs instead of the per-spec solution that was used previously. Let me know if this is not desirable.